subsys/net/ip/l2/ethernet.c

```diff
@@ -139,6 +142,9 @@ static enum net_verdict ethernet_recv(struct net_if *iface,
 		NET_DBG("ARP packet from %s received",
 			net_sprint_ll_addr((u8_t *)hdr->src.addr,
 					   sizeof(struct net_eth_addr)));
+#ifdef CONFIG_NET_IPV4_AUTOCONF
+		net_ipv4_autoconf_input(iface, pkt);
```

Comment:
yes, this is basically just for conflict detection/resolution, the regular arp handling should still stay in place afterwards